src/puppet-web/firer.ts

```diff
@@ -194,7 +195,9 @@ async function checkRoomJoin(m: Message): Promise<void> {
   let inviteeContactList: Contact[] = []
 
   try {
-    if (inviter === "You've" || inviter === '你' || inviter === 'your') {
```

Comment:
I'd like to keep this line of code, instead of your new `!inviter`. 

Because I think to check more words in the message is a good way to make sure you are tracing the right message in Wechat.